### PR TITLE
Update the image and enable e2e runs

### PR DESCRIPTION
**Pull request by gnufied**
/kind cleanup

bumping hostpath driver version to check expansions tests

### CODE REVIEW

**Comment by gnufied** [comment]
cc @pohly

**Comment by gnufied** [comment]
Looks like overriding e2e version seems to have worked but as dicussed they are being skipped now. I will enable the tests.

File: .prow.sh [hunk 1]
```diff
@@ -1,5 +1,6 @@
 #! /bin/bash
 
+CSI_PROW_KUBERNETES_VERSION=latest
```
Comment (msau42) [inline .prow.sh]: Can you add a TODO, or open an issue to remove this once we add 1.16 jobs?

Comment (gnufied) [reply .prow.sh]: opened

Comment (pohly) [reply .prow.sh]: See https://github.com/kubernetes-csi/external-resizer/issues/54

**Pull request merged**